src/components/source_editor/CodeMirror.jsx

```diff
@@ -34,7 +38,11 @@ class CodeMirror extends React.Component {
                 }}
                 onFocus={onFocus}
                 onBlur={onBlur}
-                options={options}
+                readOnly={readOnly}
+                lineNumbers={false}
+                theme={androidstudio}
```

Comment:
It seems the `darcula` theme which was used previously is also available via [www.npmjs.com/package/@uiw/codemirror-theme-darcula](https://www.npmjs.com/package/@uiw/codemirror-theme-darcula). Was there a specific reason for the change to `androidstudio`?

Reply:
Yes I was looking at https://uiwjs.github.io/react-codemirror/#/theme/data/dracula and it was looking different, but now I see that there are `darcula` and `dracula`